### ISSUE TITLE
Client Id not being sent

### ISSUE THREAD

**Issue by mastermike14** [megawubs/trakt-api-wrapper#3]
When calling trakt->authorize() the client id is not being sent to trakt raising a oauth error

**Comment by mastermike14** [comment]
in the Trakt Class i have the code

```
 public function __construct($clientId, $clientSecret, $redirectUrl)
{
    $this->clientId = ClientId::set($clientId);
    $this->clientSecret = $clientSecret;
    $this->redirectUrl = $redirectUrl;
}
```

and the ClientId class 

```
class ClientId extends AbstractParameter
{

}
```

thus making the client id null because that method doesnt even exist in the code. I change the code to the supplied client id

```
    $this->clientId = $clientId;
```

And it works just fine

**Comment by megawubs** [comment]
Fixed with: 4a74f68a959ccd934453bc3f323a188df6b14b4f
